### PR TITLE
PERF-625 First tests for reading from GDoc

### PR DESCRIPTION
**Pull request by olga-andreeva-gyg**

### CODE REVIEW

**Comment by pvonr** [comment]
Hi @olga-andreeva-gyg we can connect the tests to drone so we are able to run them via `gygdev test` after this is merged in master. no blocker to move forward. Thanks

File: tests/ad_templates/api/test_generate_gdoc.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+from tests.base import BaseTestCase
+
+
+class TestGenerateGDoc(BaseTestCase):
+    FIXTURES_PATH = 'tests/ad_templates/api/fixtures/'
+    GOOGLE_CLIENT_MOCK_PATH = 'app.ad_templates.resources.spreadsheet.get_google_doc_credentials'
+
+    def _mock_gc(self, ad_part_sheet_path, eta_ad_template_path):
+        with open(os.path.join(self.FIXTURES_PATH, 'ad_optimisation_spreadsheet.json')) as data_file:
+            spreadsheet_json = json.load(data_file)
```
Comment (andrebco) [inline tests/ad_templates/api/test_generate_gdoc.py]: Why not generating a helper that would return the json file, like:
```def whatever_helper(path, name):
    with open(os.path.join(path, 'ad_optimisation_spreadsheet.json')) as data_file:
        spreadsheet_json = json.load(data_file)

spreadsheet_json = whatever_helper(self.FIXTURES_PATH, 'ad_optimisation_spreadsheet.json')```

File: tests/ad_templates/api/test_generate_gdoc.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+from app import db
+from app.ad_templates.models import AdPart, AdLabel, EtaAdTemplate, EtaAdLocation, EtaAdCustomerInterest
+from tests.base import BaseTestCase
+
+
+class TestGenerateGDoc(BaseTestCase):
+    FIXTURES_PATH = 'tests/ad_templates/api/fixtures/'
+    GOOGLE_CLIENT_MOCK_PATH = 'app.ad_templates.resources.spreadsheet.get_google_doc_credentials'
+
+    def _mock_gc(self, ad_part_sheet_path, eta_ad_template_path):
```
Comment (andrebco) [inline tests/ad_templates/api/test_generate_gdoc.py]: In the future we could have a few helpers like this `mock_google_client` on a different file so we can use on diff places.

File: tests/ad_templates/api/test_generate_gdoc.py (hunk 1 above)
Comment (olga-andreeva-gyg) [reply tests/ad_templates/api/test_generate_gdoc.py]: Good suggestion, adjusted.

**Pull request merged**